packages/next-swc/crates/next-dev-tests/tests/integration/next/error/ssr/issues/Error during SSR Rendering-74f37a.txt

```diff
@@ -4,7 +4,7 @@ PlainIssue {
     category: "rendering",
     title: "Error during SSR Rendering",
     description: "Error: Broken page (expected error)\n\nDebug info:\n- Error: Broken page (expected error)\n at Page (packages/next-swc/crates/next-dev-tests/tests/integration/next/error/ssr/input/pages/broken.tsx:2:9)\n       1 | export default function Page() {\n         |         v\n       2 +   throw new Error('Broken page (expected error)')\n         |         ^\n       3 | }\n       4 | \n  ",
-    detail: "",
+    detail: "Node.js exit code: 1",
```

Comment:
Did we forget to update these issues or did something else change here?

Reply:
trying to figure that out right now 🙃